File: sentiment-analysis/ibm-watson-natural-language-understanding/IbmWatsonNaturalLanguageUnderstandingSentimentAnalysis.java
NaturalLanguageUnderstanding service = new NaturalLanguageUnderstanding("2018-03-16", username, password);

SentimentOptions sentimentOptions = new SentimentOptions.Builder().build();

Features features = new Features.Builder()
    .sentiment(sentimentOptions)
    .build();

AnalyzeOptions parameters = new AnalyzeOptions.Builder()
    .text(text)
    .features(features)
    .build();

AnalysisResults response = service
    .analyze(parameters)
    .execute();

SentimentResult sentiment = response.getSentiment();

File: sentiment-analysis/ibm-watson-natural-language-understanding/IbmWatsonNaturalLanguageUnderstandingSentimentAnalysis.kt
// "text" is a String holding the input document.

val service = NaturalLanguageUnderstanding("2018-03-16", username, password)

val sentimentOptions = SentimentOptions.Builder().build()

val features = Features.Builder()
    .sentiment(sentimentOptions)
    .build()

val parameters = AnalyzeOptions.Builder()
    .text(text)
    .features(features)
    .build()

val response = service
    .analyze(parameters)
    .execute()

val sentiment = response.sentiment
